Add missing egress sg rule to postgres

twunderlich-grapl opened the pull request:
<!-- Thank you for your contribution to Grapl! Please take some time
to fill out this short template to help us review and understand your
PR. -->

### Which issue does this PR correspond to?
https://github.com/grapl-security/issue-tracker/issues/1011
<!-- Please provide a link to the GitHub Issue this pull request is
meant to address. -->

### What changes does this PR make to Grapl? Why?
This fixes db timeouts because while connections can reach postgres the response can never get through
<!-- Please describe the functional impact of these changes and the
rationale behind them. This will help us understand your approach to
solving the problem. -->

### How were these changes tested?

<!-- Please describe how these changes were tested. There should be
sufficient detail that reviewers can replicate your testing
procedure. If the procedure is a manual one that's OK, your
description will help us work with you to get it automated. -->

Commit:
Add missing egress sg rule to postgres

This fixes db timeouts because while connections can reach postgres
the response can never get through

christophermaier commented on pulumi/infra/postgres.py:
```diff
@@ -104,6 +104,17 @@ def __init__(
+            type="egress",
+            security_group_id=self.security_group.id,
+            from_port=postgres_port,
+            to_port=postgres_port,
```
Is this correct? I thought clients' ports would be arbitrary, high number ones.

twunderlich-grapl replied:
Yep you're correct. Adding just egress doesn't seem to work

Commit:
Open all ports for egress

Usually with egress a high port is allocated to allow sending traffic
across multiple ports.
Add a description to each rule while we're here

twunderlich-grapl commented:
After manually testing I've confirmed that this isn't necessary. Closing this
